noir-projects/aztec-nr/aztec/src/state_vars/delayed_public_mutable.nr

```diff
@@ -168,19 +168,19 @@ where
         T: Packable<N>,
     {
         // Whenever we write to public storage, we write both the value change and delay change to storage at once.
-        // We do so by wrapping them in a single struct (`SharedMutableValues`). Then we wrap the resulting struct in
+        // We do so by wrapping them in a single struct (`DelayedPublicMutableValues`). Then we wrap the resulting struct in
         // `WithHash`.
         // Wrapping in `WithHash` makes for more costly writes but it also makes private proofs much simpler because
         // they only need to produce a historical proof for the hash, which results in a single inclusion proof (as
         // opposed to 4 in the best case scenario in which T is a single field). Private shared mutable reads are
         // assumed to be much more frequent than public writes, so this tradeoff makes sense.
```

Comment:
Mention of 'shared mutable' in the comments. I'd search the entire repo for 'sharedmutable', 'shared mutable' and 'shared_mutable' to check there's no renmnants.